server/queries/studentQuery.js

```diff
@@ -0,0 +1,24 @@
+// query for student info by email
+const mysqlConnection = require("../mysql/config");
+
+const getStudentQuery = async (email) => {
+  return new Promise((resolve, reject) => {
```

Comment:
rename to getStudentByEmail so it will be clear what the function does

Reply:
Done